src/cms/models/regions/region.py

```diff
@@ -158,6 +158,15 @@ class Region(models.Model):
         ),
     )
 
+    administrative_division_included = models.BooleanField(
+        default=False,
+        verbose_name=_("include administrative division into name"),
+        help_text=_(
+            "Determines whether the administrative division is displayed next to the region name.\
+            Sorting is always based on the name, independently from the administrative division."
```

Comment:
```suggestion
        help_text=__(
            _("Determines whether the administrative division is displayed next to the region name."),
            _("Sorting is always based on the name, independently from the administrative division."),
```
I think using the lazy formatting method `ugettext_many_lazy()` makes it a bit more readable...